back/engines/commercial/analysis/spec/services/input_to_text_spec.rb

```diff
@@ -87,6 +87,23 @@
       input = build(:idea, body_multiloc: { en: 'This is a way too long sentence!' })
       expect(service.execute(input, truncate_values: 20)).to include({ 'Description' => 'This is a way too...' })
     end
+
+    it 'includes the other field' do
+      custom_field = create(:custom_field_select, title_multiloc: { en: 'What\'s your favourite option?' })
+      create(:custom_field_option, custom_field: custom_field, key: 'other', title_multiloc: { 'en' => 'Other' }, other: true)
+      service = described_class.new([custom_field])
+      input = build(
+        :idea,
+        custom_field_values: {
+          custom_field.key => 'other',
+          "#{custom_field.key}_other" => 'Because none of the above'
+        }
+      )
+      expect(service.execute(input)).to eq({
+        'What\'s your favourite option?' => 'Other',
+        'Type your answer' => 'Because none of the above'
```

Comment:
Where is the `Type your answer` coming from exactly?

I don't think that for the LLM, this will be clear enough that's a clarification on the other option. Can we change this to something like `Since you picked 'other', what are you thinking of?`

Reply:
@kogre It's coming from here: https://github.com/CitizenLabDotCo/citizenlab/blob/48786d41b452bdc777580d0f74945b9ee1e90339/back/spec/fixtures/locales/en.yml#L62

I would propose to update the copy in this case, is it okay if it appears like this as well in the survey form?